src/mtx/commands/uplink/GetEnergyDayPreviousResponse.ts

```diff
@@ -0,0 +1,110 @@
+import Command, {TCommandExampleList, COMMAND_HEADER_SIZE} from '../../Command.js';
+import CommandBinaryBuffer from '../../CommandBinaryBuffer.js';
+import {READ_ONLY} from '../../constants/accessLevels.js';
+import {UPLINK} from '../../../constants/directions.js';
+import {TInt32} from '../../../types.js';
+import {IDate} from '../../utils/dateTime.js';
+
+
+interface IGetEnergyDayPreviousResponseParameters {
+    date: IDate,
+    /** active A+ energy by tariffs T1-T4 */
+    energy: Array<TInt32>
```

Comment:
could we use `energies ` instead `energy`

Reply:
renamed